<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeProps</class>
 <widget class="QWidget" name="NodeProps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="nodePropertiesTab">
      <attribute name="title">
       <string>Pooperties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="propertiesLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nodeIdLabel">
           <property name="text">
            <string>Node ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="nodeId">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="prefix">
            <string>0x</string>
           </property>
           <property name="maximum">
            <number>2000000000</number>
           </property>
           <property name="displayIntegerBase">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nodeFirmwareLabel">
           <property name="text">
            <string>Node firmware</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="nodeFirmware">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="nodeTagLabel">
           <property name="text">
            <string>Tag</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="nodeTag"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="logDestinationLabel">
           <property name="text">
            <string>Log destination</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="logDestination">
           <property name="prefix">
            <string>0x</string>
           </property>
           <property name="maximum">
            <number>2000000000</number>
           </property>
           <property name="displayIntegerBase">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="rebootNodeButton">
           <property name="text">
            <string>Reboot Node</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>122</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="applyHorizontalLayout">
         <item>
          <spacer name="applyHorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="applyButton">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nodeLinksTab">
      <attribute name="title">
       <string>Links</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="linksTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="servicesTab">
      <attribute name="title">
       <string>Services / Entities</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="entitiesTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="entitiesSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="entitiesRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="firmwareUpdateTab">
      <attribute name="title">
       <string>Firmware update</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="firmwarFileLabel">
           <property name="text">
            <string>Firmware file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firmwareFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="firmwareFileSelect">
           <property name="text">
            <string>Select file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QProgressBar" name="firmwareUploadState">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="firmwareUploadButton">
           <property name="text">
            <string>Upload firmware</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>266</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="workerStatusLayout">
     <item>
      <widget class="QLabel" name="workerStatusLabel">
       <property name="text">
        <string>Network worker:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="workerResult">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="workerStatus">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
